tests/kernel/update_kernel.pm

```diff
@@ -378,14 +380,17 @@ sub update_kgraft {
         # warm up system
         sleep 15;
 
-        zypper_call("in -l -t patch $patches", exitcode => [0, 102, 103], log => 'zypper.log', timeout => 2100);
+        zypper_call("in -l -t patch $patches", exitcode => [0, 102, 103], log => 'zypper.log', timeout => 2100) if is_sle;
+        trup_call('pkg in kernel-livepatch-$(uname -r | sed s/\\\./_/g)') if is_sle_micro;
```

Comment:
I'd prefer `if (is_sle_micro) ... else ...` instead of creating a code path that may do nothing (not SLE, not Micro).

Reply:
New VR: https://openqa.suse.de/tests/overview?distri=sle-micro&version=6.0&groupid=562&build=H.38.1